Profile Folder/chrome/styles/partials/content/firefox/pages/_eras.scss

```diff
@@ -1,3 +1,5 @@
+@import "chrome-0/global/final"

```

Comment:
I would recommend moving this to either `chrome-1` or `chrome-11` folder for now, I don't think there are any plans on adding `chrome-0`.